src/ReleaseTests/ProperMethodCategorizationTest.class.st

```diff
@@ -131,11 +133,16 @@ ProperMethodCategorizationTest >> testInstanceSideInitializeMethodNeedsToBeInIni
 	violating := OrderedCollection new.
 	instanceSideInitializeMethods := OrderedCollection new.
 
-	Object allSubclassesDo: [:cls | cls methods do: [:m | (m selector = #initialize and: [ m isClassSide not  ]) ifTrue: [ instanceSideInitializeMethods add: m ]]].
+	Object allSubclassesDo: [:cls | 
+		cls methods do: [:method | 
+			(method selector = #initialize and: [ method isClassSide not and: [( method package name beginsWith: 'SmalltalkCI') not ] ])
+				ifTrue: [ instanceSideInitializeMethods add: method ]]].
 
 	violating := instanceSideInitializeMethods select: [:m | m protocol ~= #initialization ].
 
-	self assert: violating size <= 4 description: 'Instance side #initialize methods should be in "initialization" method category'
+	self 
+		assert: violating size <= 4 
+		description: 'Instance side #initialize methods should be in "class initialization" method category:' , violating asString
```

Comment:
I think there is a type. It shouldn't be the category "initialization" instead of "class initialization" since we are checking instance side initialize methods?

Reply:
yes, there is a typo. thank you!